pkgs/applications/virtualization/qemu/default.nix

```
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchurl, fetchpatch, python3Packages, zlib, pkg-config, glib, overrideSDK, buildPackages
 , pixman, vde2, alsa-lib, flex, pcre2
-, bison, lzo, snappy, libaio, libtasn1, gnutls, nettle, curl, dtc, ninja, meson
+, bison, lzo, snappy, libaio, libtasn1, gnutls, nettle, curl, dtc, ninja, meson, perl
```

```suggestion
, bison, lzo, snappy, libaio, libtasn1, gnutls, nettle, curl, dtc, ninja, meson
```

I don't follow; why are we adding `perl` when the package works fine without it?

---

I caught this problem by settings `nixpkgs.config.strictDepsByDefault` to true. There is more info in this issue: https://github.com/NixOS/nixpkgs/issues/178468

For why this happens to qemu specifically, my assumption is one of its dependencies propagates buildInputs/nativeBuildInputs incorrectly, but it's not surfaced when `strictDeps` is false.